source/_patterns/02-molecules/vue-widget/__tests__/integration/vue-clock.spec.js

```diff
@@ -0,0 +1,33 @@
+import { render } from 'vue-testing-library';
+import vueClock from 'molecules/vue-widget/src/vue-clock/vue-clock.vue';
+
+describe('vue-clock.vue', () => {
+  it('sets rotate property for the seconds hand', () => {
+    const { getByTestId } = render(vueClock);
+    const secondsHand = getByTestId('seconds-hand');
+    expect(secondsHand.style.transform).toContain('rotate');
+  });
+
+  it('sets rotate property for the minutes hand', () => {
+    const { getByTestId } = render(vueClock);
+    const minutesHand = getByTestId('minutes-hand');
+    expect(minutesHand.style.transform).toContain('rotate');
+  });
+
+  it('sets rotate property for the hours hand', () => {
+    const { getByTestId } = render(vueClock);
+    const hoursHand = getByTestId('hours-hand');
+    expect(hoursHand.style.transform).toContain('rotate');
+  });
+
+  it('updates the seconds hand', done => {
+    const { getByTestId } = render(vueClock);
+    const secondsHand = getByTestId('seconds-hand');
+    const before = secondsHand.style.transform;
+    setTimeout(() => {
```

Comment:
I think this probably is a good use case for using [Jest's fake timers](https://jestjs.io/docs/en/timer-mocks.html) to handle these assertions since we can manually advance the timer. Happy to help take a look at it as well. This should also eliminate the need for the done callback as well.

Reply:
Ah, yeah, that would be great, didn't know that was a thing!

Reply:
So, after meeting with Daniel, it looks like because the element uses Date.now to keep track of time, the jest timing mocks don't work on it. Going to keep this as is for now.